python/ambassador/utils.py

```diff
@@ -348,19 +362,23 @@ def from_dict(cls, resource: 'IRResource',
                 return None
 
             cert = None
+        elif secret_type == 'istio.io/key-and-cert':
+            resource.ir.logger.error(f"{resource.kind} {resource.name}: found data but handler for istio key not finished yet")
```

Comment:
I wasn't sure if this was necessary but it didn't feel right to simply ignore this type. I didn't find that I had to add this but I'm also not exactly sure how this method is used. 🤔

Reply:
`secret_info_from_k8s` turns out to be unused, so we're good to go. ;)